mmpose/datasets/datasets/body3d/body3d_semi_dataset.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from torch.utils.data import Dataset
+
+from mmpose.datasets.builder import DATASETS, build_dataset
+
+
+@DATASETS.register_module()
+class Body3DSemiDataset(Dataset):
```

Comment:
The `Semi` in `SemiDataset` feels like partial/pseudo/quasi/incomplete. Does semi-supervised have any official abbreviations? Or we use the full version `Body3DSemiSupervisedDataset`

Reply:
It seems that "semi-supervised" doesn't have an official abbreviation. The full version is better.